docs/docs/upgrading/migrate_v5_v6.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 1
+---
+
+# Upgrading to ICS v6.x from v5.x
+
+ICS specific changes are outlined below.
+
+Pre-requisite version for this upgrade: any from the `v5.x` release line.
+
+## Note
+
+## Provider
+
+### Migration (v5 -> v6)
+
+ConensusVersion was bumped to `6`.
```

Comment:
`ConsensusVersion` on the provider seems [to be 8](https://github.com/cosmos/interchain-security/blob/7b481bd1ead9d8551a2b2790b350f2aab6b24e62/x/ccv/provider/module.go#L170). So probably in `v6` we jump to 9.